flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -2488,9 +2562,30 @@ class FMCMainDisplay extends BaseAirliners {
 
     //TODO: fix this functionality
     trySetPreSelectedCruiseSpeed(s) {
+        const isNextPhase = this.currentFlightPhase === FmgcFlightPhases.CLIMB;
```

Comment:
No entry once in cruise.
<img width="697" alt="image" src="https://user-images.githubusercontent.com/9995998/116978022-43e70a00-ad17-11eb-8384-f6e5edad56b0.png">
that note...